scrapi/processing/cassandra.py

```diff
@@ -0,0 +1,101 @@
+import json
+from uuid import uuid4
+
+from cqlengine import columns, Model, connection
+from cqlengine.management import sync_table, create_keyspace
+
+from scrapi.processing.base import BaseProcessor
+
+connection.setup(['127.0.0.1'], 'scrapi')
+create_keyspace('scrapi', replication_factor=1, strategy_class='SimpleStrategy')
+
+class CassandraProcessor(BaseProcessor):
+    NAME = 'cassandra'
+
+    def __init__(self):
+        sync_table(DocumentModel)
+        sync_table(VersionModel)
+
+    def process_normalized(self, raw_doc, normalized):
+        nm = self.send_to_database(
+            docID=normalized.get("id")['serviceID'],
+            source=normalized.get('source'),
+            url=normalized.get('id')['url'],
+            contributors=json.dumps(normalized.get('contributors')),
+            id=normalized.get('id'),
+            title=normalized.get('title'),
+            tags=normalized.get('tags'),
+            dateUpdated=normalized.get('dateUpdated'),
+            properties=json.dumps(normalized.get('properties'))
+        )
+        nm.save()
+
+    def process_raw(self, raw_doc):
+        rm = self.send_to_database(**raw_doc.attributes)
+        rm.save()
+
+    def send_to_database(self, docID, source, **kwargs):
+        documents = DocumentModel.objects(docID=docID, source=source)
+        if documents:
+            document = documents[0]
+            # Create new version, get UUID of new version, update
+            versions = document.versions
```

Comment:
I preferred that to always referencing documents[0], and I thought the name change made sense because it is a transition from a list to a single element of that list.